packages/discord.js/typings/index.d.ts

```diff
@@ -3196,9 +3196,9 @@ export function makeError(obj: MakeErrorOptions): Error;
 export function makePlainError(err: Error): MakeErrorOptions;
 export function mergeDefault(def: unknown, given: unknown): unknown;
 export function moveElementInArray(array: unknown[], element: unknown, newIndex: number, offset?: boolean): number;
-export function parseEmoji(text: string): { animated: boolean; name: string; id: Snowflake | null } | null;
+export function parseEmoji(text: string): PartialEmoji | null;
 export function resolveColor(color: ColorResolvable): number;
-export function resolvePartialEmoji(emoji: EmojiIdentifierResolvable): Partial<APIPartialEmoji> | null;
+export function resolvePartialEmoji(emoji: Emoji | EmojiIdentifierResolvable): PartialEmoji | PartialEmojiOnlyId | null;
```

Comment:
I feel like you can overload this to reflect
 > Suppling a snowflake yields `PartialEmojiOnlyId`.
 
 from the documentation

Reply:
Resolved with 67ba904f613304f2acac81b9f9942ac6084b94dc.